tests/site_nav.js

```diff
@@ -79,8 +75,8 @@ describe('SiteNav', function() {
       expect(this.$subLists.last().attr('aria-hidden')).to.equal('true');
     });
 
-    it('should assign aria-haspopup to all links to the sub lists', function() {
-      var $link = this.$fixture.find('#site-menu .js-nav-drop-link');
+    it('should assign aria-haspopup to all toggle buttons for the sub lists', function() {
+      var $link = this.$fixture.find('.js-sublist-toggle');
```

Comment:
Makes sense. Testing philosophy question though: so in this case, there is only one toggle, but in the real world there may be more. Do you write tests for what you could encounter or for how you've set up the test DOM? (Does that even make sense?)